tests/python_functional/src/syslog_ng_config/statements/sources/example_msg_generator_source.py

```diff
@@ -27,4 +27,4 @@ class ExampleMsgGeneratorSource(SourceDriver):
     def __init__(self, **options):
         self.driver_name = "example_msg_generator"
         self.DEFAULT_MESSAGE = "-- Generated message. --"
-        super(ExampleMsgGeneratorSource, self).__init__(None, options)
+        super(ExampleMsgGeneratorSource, self).__init__(logger_name=__name__, options=options)
```

Comment:
I understand the change set behind this: to pass __name__ through the classes via logger_name parameter, but I did not like this. Previously Light uses a similar form, but with that it was too difficult to read/understand why such parameter is needed or passed in certain classes. It would be a step back for Light. If you really want to identify the parent logger with the child's name let's find out other solution, I will help in this.

Reply:
I removed this.